airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSourceOperations.java

```diff
@@ -191,8 +192,17 @@ public MysqlType getDatabaseFieldType(final JsonNode field) {
         // When CHAR[N] and VARCHAR[N] columns have binary character set, the returned
         // types are BINARY[N] and VARBINARY[N], respectively. So we don't need to
         // convert them here. This is verified in MySqlSourceDatatypeTest.
+        case DECIMAL -> {
+          if (field.get(INTERNAL_DECIMAL_DIGITS) != null && field.get(INTERNAL_DECIMAL_DIGITS).asInt() == 0) {
+            return BIGINT;
+          }
+        }
+        case DECIMAL_UNSIGNED -> {
+          if (field.get(INTERNAL_DECIMAL_DIGITS) != null && field.get(INTERNAL_DECIMAL_DIGITS).asInt() == 0) {
+            return BIGINT_UNSIGNED;
+          }
+        }
```

Comment:
@nguyenaiden 
This is the only thing I think you'll need to put in postgres or any other connector in order to resolves the similar issues. (plus testing)